css-layout-classes/styles.css

```diff
@@ -0,0 +1,60 @@
+.container {
+  justify-content: center;
+  margin: auto;
+}
+
+.row {
+  padding: 5px;
+  display: flex;
+  flex-direction: row;
+  max-width: 800px;
+}
+
+div {
+  margin: auto;
+  max-width: 800px;
+}
+
+header {
+  margin: auto;
+  max-width: 800px;
+  width: 100%;
+}
+
+img {
+  margin: auto;
+  max-width: 800px;
+  width: 100%;
+}
+
+.text-column {
+  flex: 2;
+}
+
+.picture-column {
+  flex: 1;
+}
+
+.dk-column {
+  flex: 1;
+}
+
+.mario-column {
+  flex: 1;
+}
```

Comment:
Since these are the same width, try making just one class for them that takes up 1/3rd the width of their parent.